Core/PrivacyConfiguration.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  PrivacyConfiguration.swift
+//  DuckDuckGo
+//
+//  Copyright © 2021 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+public struct PrivacyConfiguration: Codable {
```

Comment:
Since we are embedding configuration now, along with etag, we should add unit tests that:
- Validate if etag matches embedded file using SHA - see `testWhenEmbeddedDataIsUpdatedThenUpdateSHAAndEtag()` for reference.
- Validate if embedded config is what we expect e.g. if tracker blocking and gpc is enabled.